### ISSUE TITLE
`value-fixture` contains version 61 .class files (should be version 52) (2.10.0)

### ISSUE THREAD

**Issue by snazy** [immutables/immutables#1486]
Builds that use `org.immutables:value-fixture` and run with/for Java 11 or 8 fail with immutables v2.10.0.

The class files in `value-fixture-2.10.0.jar` are all compiled for Java 17 (class file version 61). I suspect, the class files should be compiled for Java 8 as documented (["`... use @Nullable, @AllowNulls, @SkipNulls as Java 8 type annotation...`"](https://immutables.github.io/immutable.html#nulls-in-collection)).

Immutables 2.9.3 was fine.

**Comment by elucash** [comment]
is it a problem with "value-fixture" only? or other modules are affected. If this is only "value-fixture", then it can be fixed (haven't looked in those profiles in a while), but should not an issue as this is test-only module, hope no one is really using it.

**Comment by snazy** [comment]
Yes, only that artifact is affected.
Sadly, that artifact is required to use the `@AllowNulls` + `@SkipNulls` annotations.

**Comment by elucash** [comment]
Thats not true, this is test only artifact, noone should include it code. Those annotations you mention should be your own, create them or copy paste in your package. These annotations are bring-your-own-annotations so only simple name should match!
